frontend/src/app/settings/modelsources/t4c-settings/edit-t4c-instance/edit-t4c-instance.component.spec.ts

```diff
@@ -1,27 +0,0 @@
-/*
```

Comment:
Why removing this file?

Reply:
I'm not sure. I removed some files for my previous PR (persistent sessions), but not for this PR as far as I can remember.

Reply:
I undid the removal.